presto-docs/src/main/sphinx/release/release-342.md

```diff
@@ -0,0 +1,77 @@
+# Release 342 (XX Sep 2020)
+
+## General Changes
+
+* Add {func}`from_iso8601_timestamp_nanos` function. ({issue}`5048`)
+* Improve performance of queries that use the `DECIMAL` type. ({issue}`4886`)
+* Improve performance of queries involving `IN` with subqueries by extending support for dynamic
+  filtering and dynamic partition pruning. ({issue}`5017`)
+* Improve performance and latency of queries leveraging dynamic partition pruning with dynamic filters. ({issue}`4988`)
+* Improve performance of queries joining tables with missing or incomplete column statistics when
+  the cost-based optimizer is enabled (which is the default). ({issue}`5141`)
+* Reduce latency for queries where broadcast join is used and broadcasted table is large. ({issue}`5237`)
+* Fix query failure when lambda expression references a table column containing a dot. ({issue}`5087`)
+
+## Atop Connector Changes
+
+* Fix incorrect query results when query contains predicates on `start_time` or `end_time` column. ({issue}`5125`)
+
+## Elasticsearch Connector Changes
+
+* Allow reading boolean values stored as strings. ({issue}`5269`)
+
+## Hive Connector Changes
+
+* Add support for S3 encrypted files. ({issue}`2536`)
+* Add support for ABFS OAuth authentication. ({issue}`5052`)
+* Support reading timestamp with microsecond or nanosecond precision. This can be enabled with `hive.timestamp-precision`
+  connector configuration property. ({issue}`4953`)
+* Allow specifying what happens if data is inserted into existing Hive partition. 
+  This can be done using `hive.insert-existing-partitions-behavior` config property. ({issue}`4999`)
+* Add property (`hive.dynamic-filtering-probe-blocking-timeout`) for delaying table scans
```

Comment:
> Allow delaying table scans until dynamic filtering can be performed more efficiently. This can be enabled using the xxx configuration property or the xxx session property.